examples/opendsme/Makefile

```diff
@@ -0,0 +1,46 @@
+# name of your application
+APPLICATION = opendsme_example
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= nrf52840dk
+
+# Whitelist nrf802154-based boards.
+BOARD_WHITELIST := nrf52dk nrf52840dk nrf52840dongle
```

Comment:
Clarification question: Is it really only working for nrf802154 at the moment?

Reply:
yes, because the other radios are not configured to receive beacons. It should be simple to add though

Reply:
Then better add a comment as to *why* those are whitelisted.

Reply:
done!